pex/commands/bdist_pex.py

```diff
@@ -122,7 +123,7 @@ def run(self):
         log.info('Writing environment pex into %s' % target)
 
       log.debug('Building pex via: {}'.format(' '.join(cmd)))
-      process = Executor.open_process(cmd, env=env)
+      process = Executor.open_process(cmd, stderr=subprocess.PIPE, env=env)
       _, stderr = process.communicate()
```

Comment:
In Py3, this will print on line 130 with the `b''` prefix rendered in the final output, as `stderr` will be a bytes stream. To fix, we would have to add `decode('utf-8')`.

Feel free to ignore as this issue was there before this PR and it won't break anything as far as I can tell. (It won't render things like `\n` properly, however.)

Reply:
Thanks - fixed.